File: 100 problem/9average_of_all_elements.cpp
/********************************************************************
 * WRITE A PROGRAM TO FIND THE AVERAGE OF ALL ELEMENTS IN AN ARRAY. *
 ********************************************************************/


#include<iostream>
#include<vector>
using namespace std;


int average_of_all_elements(vector<int>arr) {
    int sum = 0;

    for(auto& i : arr) sum += i;

    int avg = sum / arr.size();

    return avg;
}


int main() {
    vector<int>arr = {10, 20, 30, 40, 50};

    cout << average_of_all_elements(arr);
}

File: 100 problem/9average_of_all_elements.js
/********************************************************************
 * WRITE A PROGRAM TO FIND THE AVERAGE OF ALL ELEMENTS IN AN ARRAY. *
 ********************************************************************/

function average_of_all_elements(arr) {
    let sum = 0;

    for ( let i = 0; i < arr.length; i++) {
        sum += arr[i];
    }

    let avg = sum / arr.length;
    return avg;
}


arr = [ 10, 20, 30, 40, 50]

console.log(average_of_all_elements(arr))